hull.go

```diff
@@ -0,0 +1,100 @@
+package jujusvg
+
+import (
+	"image"
+	"math"
+	"sort"
+)
+
+// getPointOutside returns a point that is outside the hull of existing placed vertices so that an object can be placed on the canvas without overlapping others.
```

Comment:
can you please limit the line length of comments to around about 70-80 characters?